cloudalerts/cloud_loggers/log_filter.py

```diff
@@ -0,0 +1,28 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at https://mozilla.org/MPL/2.0/.
+
+CENSORED_EVENT_VALUES_BY_EVENT_KEY = {
+    "headers": ["Authorization", "X-Forwarded-For"],
+    "multiValueHeaders": ["Authorization"],
+}
+
+
+def censor_event_dict(event_dict):
+    for k, v in event_dict.items():
+        if isinstance(v, dict):
+            censor_event_dict(v)
+        else:
+            for event_key, event_values in CENSORED_EVENT_VALUES_BY_EVENT_KEY.items():
+                if event_dict is not None:
```

Comment:
wouldn't you know event_dict is not None because you are already iterating over it?

Reply:
Correct sir, I have updated accordingly.